cli/utils.js

```diff
@@ -194,9 +194,11 @@ const createYeomanEnv = () => {
     Object.keys(SUB_GENERATORS)
         .filter(command => !SUB_GENERATORS[command].cliOnly)
         .forEach(generator => {
-            if (SUB_GENERATORS[generator].blueprint) {
-                /* eslint-disable prettier/prettier */
-                env.register(require.resolve(`${SUB_GENERATORS[generator].blueprint}/generators/${generator}`), `${CLI_NAME}:${generator}`);
+            if (SUB_GENERATORS[generator].blueprints && SUB_GENERATORS[generator].blueprints.length > 0) {
```

Comment:
Not sure if this should be changed to `blueprints` and iterated. AFAIK this flag is set on a custom command coming from any blueprint so the actual merge needs to be done in `commands.js` and not here. This would not work i think

Reply:
Thanks. Let me re-check

Reply:
You are right. It was not needed and it has been removed.

Reply:
Thanks. You still need to update commands.js to work with the new flag as it still has some code for blueprint stuff

Reply:
I think this should not be changed as it does not make sense to call the same sub-command for multiple blueprints or am I missing something?

Reply:
Looking again at `commands.js` it simply registers commands instead of calling them, my bad. So you are right that it should be changed. What I fail to see though is where the `blueprint` flag is being set in order for it to be used at `utils.createYeomanEnv()`. Maybe @sahbi-ktifa  can help who I believe coded this part?

Reply:
I think the blueprint flag is set by the blueprints itself when defining its own command.js file which is merged here

Reply:
OK. Thanks. Fixed `commands.js` to register commands from multiple blueprints